apache-Dockerfile-block-1

```diff
@@ -5,7 +5,7 @@ RUN rm -rf /var/www/html && mkdir -p /var/lock/apache2 /var/run/apache2 /var/log
 # Apache + PHP requires preforking Apache for best results
 RUN a2dismod mpm_event && a2enmod mpm_prefork
 
-RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist
+RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist && rm /etc/apache2/{conf,sites}-enabled/*
```

Comment:
I agree on keeping the distribution configuration around, but I'd stick with deleting the links. That's what folks would expect from an Apache web server on Debian (and derivatives), deleting the links is the same as calling `a2disconf`/`a2dissite` to disable configuration. Disabled configuration stays in `conf.available`, active configuration is symlinked to the `conf.enabled` folder.